### ISSUE TITLE
atime and mtime not preserved.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [XianglinLi/opendedup#7]
```
SDFS 0.8.11, Debian.

 hypatia:/backup/current# touch -d yesterday testfile
 hypatia:/backup/current# stat testfile
   File: `testfile'
   Size: 0               Blocks: 0          IO Block: 4096   regular empty file
 Device : 12h/18d Inode: 11981       Links: 1
 Access: (0644/-rw-r--r--)  Uid: (    0/    root)   Gid: (    0/    root)
 Access: 2010-03-30 15:21:02.000000000 +0200
 Modify: 2010-03-30 15:21:02.000000000 +0200
 Change: 2010-03-30 15:21:02.000000000 +0200
 hypatia:/backup/current# date
 Tue Mar 30 15:21:10 CEST 2010
 hypatia:/backup/current# touch -d yesterday /tmp/testfile
 hypatia:/backup/current# ls -l /tmp/testfile
 -rw-r--r-- 1 root root 0 Mar 29 15:21 /tmp/testfile
 hypatia:/backup/current# mv -f /tmp/testfile .
 hypatia:/backup/current# ls -l testfile
 -rw------- 1 root root    0 Mar 30 15:21 testfile

```

Original issue reported on code.google.com by `stur...@gmail.com` on 30 Mar 2010 at 1:25

**Comment by GoogleCodeExporter** [comment]
```
I am pretty sure I fix this in 0.8.12
```

Original comment by `sam.silv...@gmail.com` on 31 Mar 2010 at 7:09
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
Original comment by `sam.silv...@gmail.com` on 1 Apr 2010 at 6:10
- Changed state: **Fixed**
